hub/core/transform/transform.py

```diff
@@ -98,6 +93,11 @@ def eval(
         if isinstance(data_in, hub.core.dataset.Dataset):
             data_in = get_dataset_with_zero_size_cache(data_in)
 
+        hub_reporter.feature_report(
+            feature_name="eval",
+            parameters={"Num_Workers": num_workers, "Scheduler": scheduler},
```

Comment:
@istranic mypy (which is doing type inference) was warning that `num_workers` is of type int however expects str, should we wrap `str(num_workers)`